connectors/src/connectors/notion/index.ts

```diff
@@ -107,32 +97,24 @@ export async function stopNotionConnector(
 }
 
 export async function resumeNotionConnector(
-  dataSourceConfig: DataSourceConfig,
-  nangoConnectionId: string
+  connectorId: string
 ): Promise<Result<string, Error>> {
-  const connector = await Connector.findOne({
-    where: {
-      type: "notion",
-      workspaceId: dataSourceConfig.workspaceId,
-      dataSourceName: dataSourceConfig.dataSourceName,
-    },
-  });
+  const connector = await Connector.findByPk(connectorId);
 
   if (!connector) {
     logger.error(
       {
-        workspaceId: dataSourceConfig.workspaceId,
-        dataSourceName: dataSourceConfig.dataSourceName,
+        connectorId: connectorId,
```

Comment:
No error object to pass here. SHould I pass an `new Error("Notion connector not found.")` ?